src/components/Comments.jsx

```diff
@@ -21,6 +29,22 @@ function Comments({ article_id }) {
 		'Nov',
 		'Dec',
 	]
+	function handleSubmit(event) {
+		event.preventDefault()
+		setIsCommenting(true)
+		if (newComment.length >= 5) {
+			addComment(article_id, {
+				body: newComment,
+				author: activeUser.username,
+			}).then((data) => {
+				setComments((currValue) => {
+					return [data, ...currValue]
+				})
+				setIsCommenting(false)
+				setNewComment('')
```

Comment:
One issue though. If the comment posting is unsuccessful, the user isn't given any feedback. I would again harness the catch block to display an error message to the user if the posting comment is unsuccessful